cmd/operator-sdk/bundle/cmd.go

```diff
@@ -53,9 +54,17 @@ type bundleCmd struct {
 // cleanupFuncs returns a set of general funcs to clean up after a bundle
 // subcommand.
 func (c bundleCmd) cleanupFuncs() (fs []func()) {
-	metaDir := filepath.Join(c.directory, bundle.MetadataDir)
-	dockerFile := filepath.Join(c.directory, bundle.DockerFile)
+	manifestDir := c.outputDir
+	if manifestDir == "" {
+		manifestDir = c.directory
+	}
+	absManifestDir, _ := filepath.Abs(manifestDir)
```

Comment:
Does this need to be abs? If so import `log "github.com/sirupsen/logrus"` above and add:

```suggestion
	absManifestDir, err := filepath.Abs(manifestDir)
	if err != nil {
		log.Fatal(err)
	}
```